lib/transition.js

```diff
@@ -294,6 +308,22 @@ class Transition {
       }
       return undefined
     }
+
+    /**
+     * Check if transition prop is inside vendor specific rule
+     */
+    inVendorSpec (decl) {
```

Comment:
`*Spec` reminds me test’s spec. I think `ruleVendorPrefixes()` will be better name (and we return `declPrefixes` here)

Reply:
Yes, it's more clear. Renamed it.